src/SWP/Bundle/CoreBundle/Model/FacebookPage.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Model;
+
+use SWP\Bundle\FacebookInstantArticlesBundle\Model\Page;
+use SWP\Component\MultiTenancy\Model\TenantAwareInterface;
+use SWP\Component\MultiTenancy\Model\TenantAwareTrait;
+use SWP\Component\Storage\Model\PersistableInterface;
+
+class FacebookPage extends Page implements TenantAwareInterface, PersistableInterface
```

Comment:
then I wont be able (for example) to use FBIABundle in other project and add, remove page if I would use page repository from FBIABundle I would need to provide my custom implementation just to implement Persistable interface while it can be used in PageInterface directly..

FBIABundle uses StorageBundle already so `PageInterface` should extend `PersistableInterface` in FBIABundle.

and +1 for not creating empty interface - no needed here, I agree